src/tpm2_wrap.c

```diff
@@ -100,8 +100,13 @@ static int wolfTPM2_Init_NoDev(TPM2_CTX* ctx, TPM2HalIoCb ioCb, void* userCtx,
 int wolfTPM2_Test(TPM2HalIoCb ioCb, void* userCtx, WOLFTPM2_CAPS* caps)
 {
     int rc;
+    TPM2_CTX* current_ctx;
     TPM2_CTX ctx;
 
+    /* Backup active TPM context */
+    current_ctx = TPM2_GetActiveCtx();
```

Comment:
Does it make sense to validate the return value here?

Reply:
No need to check. NULL is okay or any pointer is okay.